skrub/tests/test_fuzzy_join.py

```diff
@@ -313,9 +314,51 @@ def test_numerical_column() -> None:
     assert fj_num3.shape == (2, n_cols)
 
 
-def test_multiple_keys() -> None:
+def test_datetime_column():
+    """
+    Testing that fuzzy_join works with datetime columns.
+    """
+
+    left = pd.DataFrame(
```

Comment:
In addition to testing the shape, we probably should check that "09/10/2022" is fuzzily joined with "10/10/2022".
```python
from pandas.testing import assert_frame_equal

assert_frame_equal(fj_time, fj_time_expected)
```
should do the trick!